array_api_compat/common/_linalg.py

```diff
@@ -139,7 +139,7 @@ def matrix_norm(
     xp: Namespace,
     *,
     keepdims: bool = False,
-    ord: float | Literal["fro", "nuc"] | None = "fro",
+    ord: JustInt | JustFloat | Literal["fro", "nuc"] | None = "fro",
```

Comment:
can we tighten to this?

```suggestion
    ord: Literal[1, 2, -1, -2] | JustFloat | Literal["fro", "nuc"] | None = "fro",
```